src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -14,30 +14,615 @@
  * limitations under the License.
  */
 
-export const human38stats = {
-  assembly_stats: {
-    chromosome_count: 23,
-    genome_size: 3096649726,
-    total_sequence_length: 4537931177
+import { SpeciesStatsSection } from 'src/content/app/species/state/general/speciesGeneralHelper';
+
+type RawSpeciesStats = {
+  [genomeId: string]: {
+    [key in SpeciesStatsSection]: {
+      [key: string]: string | number | null;
+    };
+  };
+};
+
+export const sampleData: RawSpeciesStats = {
+  homo_sapiens_GCA_000001405_28: {
+    assembly_stats: {
+      scientific_name: 'Homo sapiens',
+      sex: null,
+      strain: null,
+      taxonomy_id: 9606,
+      assembly_name: 'GRCh38.p13',
+      assembly_accession: 'GCA_000001405.28',
+      assembly_date: '2019-02-28',
```

Comment:
Do you want to logically separate the fields that describe the assembly (scientific_name, sex, taxonomy_id, assembly_name, assembly_accession, assembly_date) into a dedicated section (`assembly_description`?, `assembly_summary`?)? It feels like a perceptibly different kind of data.

Reply:
That sounds like a good idea. `assembly_summary` feels like a better name.

Reply:
I tried to make this change but currently these sections each represents an individual `ExpandableComponent`. Creating a separate section called `assembly_summary` which will not have it's own `ExpandableComponent` feels like it will cause more confusion. Would it be worth getting back to this later on?

Reply:
> Would it be worth getting back to this later on?

I think it would be nice to have this separation by the time we give our json back to Fergal so that he can update it programmatically. Were you thinking of getting back to it earlier than that (e.g. during your work on the sidebar)?

Reply:
I've tried to do it in this PR itself. Please check if it looks okay.